letsencrypt/config.yaml

```
@@ -35,6 +35,7 @@ schema:
   challenge: list(dns|http)
   acme_root_ca_cert: str?
   acme_server: url?
+  chain: str?
```

```suggestion
  chain: list(ISRG Root X1|DST Root CA X3)?
```
Based on the conversation, these are the two options. In that case we should list them rather then making users look them up and copy and paste in or potentially mistype things.